recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "6.1.7" %}
+{% set version = "6.1.9" %}
```

Comment:
Mentioned in #50, but this version contains https://github.com/jupyter/jupyter_client/pull/596 which is going to cause other problems...